test/v1/models/request/sample/disclosures/AmendDisclosuresRequestBodySpec.scala

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
This shouldn't be in the sample folder so refactor so it's /request/disclosures.